docs/src/components/ComponentDoc/ComponentExample/ComponentExampleVariables.tsx

```diff
@@ -21,13 +21,23 @@ type ComponentExampleVariablesProps = {
   usedVariables: Record<string, string[]>
 }
 
+const getGroupName = (variableName: string): string => {
+  if (/^(font|letter|line)/i.test(variableName)) return 'Typography'
+  if (/color/i.test(variableName)) return 'Colors'
```

Comment:
Let's not show the undefined values from the color palette, because if one of them is picked, the variable disappears from the form..

Reply:
Fixed, thank you 👍